lib/ferrum/page/screenshot.rb

```diff
@@ -159,6 +173,17 @@ def maybe_resize_fullscreen(full)
       ensure
         resize(width: width, height: height) if full
       end
+
+      def with_background_color(background_rgba_color)
+        if background_rgba_color
+          r, g, b, a = background_rgba_color
+          command('Emulation.setDefaultBackgroundColorOverride', color: { r: r, g: g, b: b, a: a })
```

Comment:
Then we could make it `command('Emulation.setDefaultBackgroundColorOverride', color: color.to_h)` if it's an instance of RGBA